Core/Core.cpp

```diff
@@ -95,7 +97,7 @@ void Core_Halt(const char *msg)  {
 void Core_Stop() {
 	Core_UpdateState(CORE_POWERDOWN);
 	Core_NotifyShutdown();
-	m_hStepEvent.notify_one();
+	m_StepCond.notify_one();
```

Comment:
I think "events" and condvars are slightly different, hopefully not in an important way for us... but, do we need to lock the mutex here?  It's possible that this code might run after `Core_IsActive()` but before `m_InactiveCond.wait(guard);`...

-[Unknown]

Reply:
Note that all I did was inline the contents of the event method calls, so nothing changed. As the resulting code does not look right, this exposes that the events are a broken idea :)

Fixing the bugs is out of scope for this pull requests but should be done after.

Reply:
Hmm, I thought events had used Windows events, and those tracked a flag (which is why `.reset()` existed?)  Did we change at some point?

-[Unknown]

Reply:
Events have long used condition variables on platforms other than Windows (for the Qt debugger, I guess) and a while ago I ripped out the Windows-ifdeffed path since the condition_variable one appeared to work. And it does, but it's clearly not clean and there might be some rare race conditions.